tests/test_callable.py

```diff
@@ -133,7 +134,6 @@ def test_manyfloatsum():
     for arch in ('i386', 'x86_64'):
         yield run_manyfloatsum, arch
 
-@slow_test
```

Comment:
interesting that nose2 does the filtering on the individual runners instead of the generator. Could we we leave the annotation on the generators for compatibility with nose1?

Reply:
To be honest, I'm not sure to what extent that is intended behaviour, vs just brittleness on nose2's part. Regardless, I'll leave the annotations in both places since it doesn't hurt.